src/app/ipv/tests/handleJourneyActionRequest.test.ts

```diff
@@ -140,10 +141,11 @@ describe("handleJourneyActionRequest", () => {
       ...testReq,
       session: { ...testReq.session, ipvSessionId: undefined },
     };
+
     await middleware.handleJourneyActionRequest(req, res, next);
-    expect(res.status).to.have.been.calledWith(401);
-    expect(res.render).to.have.been.calledWith("ipv/page/pyi-technical.njk", {
-      context: "unrecoverable",
-    });
+
+    expect(next).to.have.been.calledWith(
+      sinon.match.instanceOf(UnauthorizedError)
```

Comment:
There is a change here - we'll now render a 401 with a 'session ended' page, rather than a 401 with a 'technical error' page.

This feels more appropriate to me, given that the error here is ipvSessionId missing, but I am a little unsure about why this is happening in prod. It seems to happen a lot at the end of the journey - perhaps the user pressing back after being redirected to the RP?